docs/source/integrations/databricks_development_workflow.md

```diff
@@ -0,0 +1,324 @@
+
+- Auto-completion and suggestions for code, improving your development speed and accuracy.
+- Linters like Pylint or Flake8 can be integrated to catch potential issues in your code.
+- Static type checkers like Mypy can check types in your code, helping to identify potential type-related issues early in the development process.
+
+To set up these features, look for instructions specific to your IDE (for instance, [VS Code](https://code.visualstudio.com/docs/python/linting)).
+
+## Prerequisites
+
+- New or existing [AWS account](https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account/) with administrative privileges.
+- Active [Databricks deployment](https://docs.databricks.com/getting-started/index.html) on AWS (Databricks Community Edition won't suffice as it doesn't allow you to provision personal tokens).
+- A [Databricks cluster](https://docs.databricks.com/clusters/configure.html) configured with a recent version (>= 11.3 is recommended) of the Databricks runtime.
+- [Conda installed](https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html) on your local machine.
+- An account on a Git provider such as [GitHub](https://github.com/). You can check which [Git providers are supported by Databricks](https://docs.databricks.com/repos/index.html#supported-git-providers).
+- [Git installed](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) on your local machine.
+
+## Develop a project using the hybrid workflow
+
+### Create a virtual environment and install Kedro and the Databricks CLI
+
+Create a virtual environment for this tutorial using Conda:
+
+```bash
+conda create --name kedro-hybrid-workflow python=3.10
+```
+
+Once it is created, activate it:
+
+```bash
+conda activate kedro-hybrid-workflow
+```
+
+With your Conda environment activated, install Kedro and the Databricks CLI:
+
+```bash
+pip install kedro databricks-cli --upgrade
+```
+
+After installing the Databricks CLI, you must authenticate it with your [Databricks instance]([Databricks documentation](https://docs.databricks.com/dev-tools/cli/index.html#set-up-authentication).
+
+### Create a new project and add it to Version Control
+
+Create a Kedro project with the PySpark Iris starter. To do this, run the following command in your local environment:
+
+```bash
+kedro new --starter=pyspark-iris
+```
+
+Name your new project `iris-databricks` for consistency with the rest of this guide.
+
+This command will create a new Kedro project using the PySpark Iris starter template. Initialize Git for the Kedro project by running the following commands:
+
+```bash
+cd iris-databricks
+git init
+```
+
+### Add your project to Version Control
+
+Create a new private Git repository with your Git provider. Then, generate an access token to connect your Kedro project with Databricks. Look up the documentation of your Git provider to learn how to generate an access token if you are unclear, as this is different for each provider.
```

Comment:
I didn't initially realise these were separate steps. Perhaps put them on new lines to make that clear. 
I'm also not quite sure where to add my access token after I've created it? -> You've described this later on so maybe just remove this sentence here.